analyze_issue_metadata.py

```diff
@@ -0,0 +1,186 @@
+                    "%Y-%m-%dT%H:%M:%S+00:00")
+
+                # it is invalid for an issue to carry multiple priority labels
+                # so count the most severe p0 THEN p1 THEN p2+ THEN fr_question
+
+                if any(x in data_decode[repo]['issues'][issue]['labels']
+                       for x in p0_labels):
+                    analysis[repo]['p0']['count'] += 1
+                    analysis[repo]['p0']['age'] += (dt.utcnow() -
+                                                    created_date).days
+
+                elif any(y in data_decode[repo]['issues'][issue]['labels']
+                         for y in p1_labels):
+                    analysis[repo]['p1']['count'] += 1
+                    analysis[repo]['p1']['age'] += (dt.utcnow() -
+                                                    created_date).days
+
+                elif any(z in data_decode[repo]['issues'][issue]['labels']
+                         for z in p2_labels):
+                    analysis[repo]['p2+']['count'] += 1
+                    analysis[repo]['p2+']['age'] += (dt.utcnow() -
+                                                     created_date).days
+
+                elif any(x in data_decode[repo]['issues'][issue]['labels']
+                         for x in fr_question):
+                    analysis[repo]['fr_question']['count'] += 1
+                    analysis[repo]['fr_question']['age'] += (
+                        dt.utcnow() - created_date).days
+
+                else:
+                    analysis[repo]['no_priority_label']['count'] += 1
+                    analysis[repo]['no_priority_label']['age'] += (
+                        dt.utcnow() - created_date).days
+
+            if analysis[repo]['p0']['count'] > 0:
+                analysis[repo]['p0']['age'] = analysis[repo]['p0'][
+                    'age']/analysis[repo]['p0']['count']
+            if analysis[repo]['p1']['count'] > 0:
+                analysis[repo]['p1']['age'] = analysis[repo]['p1'][
+                    'age']/analysis[repo]['p1']['count']
+            if analysis[repo]['p2+']['count'] > 0:
+                analysis[repo]['p2+']['age'] = analysis[repo]['p2+'][
+                    'age']/analysis[repo]['p2+']['count']
+            if analysis[repo]['no_priority_label']['count'] > 0:
+                analysis[repo]['no_priority_label']['age'] = analysis[repo][
+                    'no_priority_label']['age']/analysis[repo][
+                    'no_priority_label']['count']
+            if analysis[repo]['fr_question']['count'] > 0:
+                analysis[repo]['fr_question']['age'] = analysis[repo][
+                    'fr_question']['age']/analysis[repo]['fr_question'][
+                        'count']
+
+    file_time = dt.now().strftime("%Y%m%d")
+
+    with open('../output_files/' + file_time +
+              '_repo_issue_analysis.csv', 'w') as csvfile:
+
+        datawriter = csv.writer(csvfile, delimiter=',', quotechar='"')
```

Comment:
Thanks.  I'm pretty sure I used this somewhere recently.  I don't know why I didn't think of it here :|